assets/scss/modules/_datatables.scss

```diff
@@ -104,7 +104,11 @@
   .sorting_desc_disabled {
     background: url("../images/sort_desc_disabled.png") no-repeat scroll center right;
   }
-
+  
+  .sorting--hidden {
```

Comment:
Part3 of redesign will need to look at modules/enhancedTables.js and enable passing of different options. For sake of incremental improvements I opted to use the css rule.